src/pages/community-champions.js

```diff
@@ -353,6 +359,90 @@ const HowToQualifyCard = ({ icon, title, description }) => (
     </HowToQualifyCardContainer>
 );
 
+const Line = styled('hr')`
+    color: ${({ theme }) => theme.colorMap.greyDarkOne};
+    margin-bottom: ${LINE_BOTTOM_MARGIN};
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const GreenBulletedListWithGreyText = styled(GreenBulletedList)`
+    li {
+        margin-bottom: ${size.xsmall};
+    }
+    p {
+        color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    }
+`;
+
+const OtherDetailsAndRequirementsContainer = styled('div')`
+    display: grid;
+    grid-template-columns: ${OTHER_DETAILS_AND_REQUIREMENTS_IMAGE_SIZE} auto;
+    @media ${screenSize.upToMedium} {
+        ul {
+            /* Show list under image instead */
+            display: none;
+        }
+    }
+`;
+
+const OtherDetailsAndRequirementsTextSection = styled('div')`
+    h5 {
+        @media ${screenSize.upToMedium} {
+            margin-bottom: ${size.tiny};
+        }
+    }
+    p {
+        max-width: ${OTHER_DETAILS_AND_REQUIREMENTS_DESCRIPTION_MAX_WIDTH};
+    }
+    ul {
+        max-width: ${OTHER_DETAILS_AND_REQUIREMENTS_LIST_MAX_WIDTH};
+    }
```

Comment:
Would it make more sense to use `styled` on each of these instead of the top div? Would also help with the refactoring